src/reducers/index.ts

```diff
@@ -1,14 +1,31 @@
 import courseReducer from "./course";
 import schedulerUtilReducer from "./schedulerUtils";
 import authReducer from "../lib/authSlice";
+import featureFlagReducer from "../lib/featureFlagSlice";
+
 import goodrichReducer from "./goodrich";
 
 const rootReducer = {
   courseState: courseReducer,
   schedulerUtilState: schedulerUtilReducer,
   authState: authReducer,
   goodrichState: goodrichReducer,
+  featureFlagState: featureFlagReducer,
 };
 
+export interface RootState {
+  courseState: ReturnType<typeof courseReducer>;
+  schedulerUtilState: ReturnType<typeof schedulerUtilReducer>;
+  authState: ReturnType<typeof authReducer>;
+  goodrichState: ReturnType<typeof goodrichReducer>;
+  featureFlagState: ReturnType<typeof featureFlagReducer>;
+}
+
```

Comment:
You should not write this manually. They are auto generated here

https://github.com/WilliamsStudentsOnline/wso-react/blob/d3e763402a11a6e41a81b3737741c3df6077eb7f/src/lib/store.ts#L71